specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -44,35 +43,85 @@ Jupyter is chosen because it is  very well-known among the data community, it is
 
 
 ## Requirements and goals
+
+### Requirements
+Legend (terms and table are based on Pragmatic Marketing recommendations):
+* Problem - a discrete pain or issue that has been observed within the target market segment. In the below table it's a short summary name of the problem.
+* Use Scenario - a description of a problem. Includes a detailed description of the typical situation that causes this problem to occur and possibly current results.
+* Evidence - the percentage of interviewed users who have mentioned the problem.
+* Impact - how much impact does the problem have on their work (high/moderate/low).
+
+|    Persona    |                 Problem                  |                                                                                                                  Use Scenario                                                                                                                  | Evidence |  Impact  |
+|:-------------:|:----------------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------:|:--------:|
+| Data Engineer |             Working with CLI             |                                                                As data engineers we do not have much experience with CLI, we need to learn how to use CLI in order to use VDK.                                                                 |   70%    |   high   |
```

Comment:
I actually prefer to remove the Persona from the table 

A) Because it's the same one so it's redundant. 
B) We have not defined the Personas anywhere in our Github documentation. Without definition to refer to, people have different assumption about what it means.